<ui version="4.0" >
 <class>FileSelectDlg</class>
 <widget class="QDialog" name="FileSelectDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Which Files You Want to Download</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1_2" >
       <property name="text" >
        <string>Download to:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" native="1" name="m_downloadLocation" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Group:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_cmbGroups" />
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="m_chkUserControlled" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;User controlled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_chkStartTorrent" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Start torrent</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QTreeWidget" name="m_file_view" >
       <property name="fullWidth" stdset="0" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>File</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Size</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="KPushButton" name="m_select_all" >
         <property name="text" >
          <string>Select &amp;All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="m_select_none" >
         <property name="text" >
          <string>Select &amp;None</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="m_invert_selection" >
         <property name="text" >
          <string>Invert Selection</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Disk space</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Required disk space:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel3" >
          <property name="text" >
           <string>Free disk space:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel6" >
          <property name="text" >
           <string>After download:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="lblRequired" >
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblFree" >
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatus" >
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="m_ok" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
       <property name="stdItem" stdset="0" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_cancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="stdItem" stdset="0" >
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_ok</tabstop>
  <tabstop>m_cancel</tabstop>
  <tabstop>m_downloadLocation</tabstop>
  <tabstop>m_cmbGroups</tabstop>
  <tabstop>m_chkUserControlled</tabstop>
  <tabstop>m_chkStartTorrent</tabstop>
  <tabstop>m_file_view</tabstop>
  <tabstop>m_select_all</tabstop>
  <tabstop>m_select_none</tabstop>
  <tabstop>m_invert_selection</tabstop>
 </tabstops>
 <includes>
  <include location="global" >kurlrequester.h</include>
  <include location="global" >klineedit.h</include>
  <include location="global" >kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_chkUserControlled</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_chkStartTorrent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
